frontend/TvCv19.Frontend/Domain/Repositories/IPatientRepository.cs

```diff
@@ -19,7 +19,7 @@ public interface IPatientRepository
     public class PocPatientRepository : IPatientRepository
     {
         private static List<Patient> _admittedPatients = new List<Patient>();
-        private static int _count;
+        private static int _count = 200;
```

Comment:
only set to 200 when working locally

Reply:
removed the 200 and set it back to original